app/public.js

```diff
@@ -269,19 +287,29 @@
       top: 0,
       left: 0
     };
-    if (elem.x && elem.y) {
-      location.top = elem.y;
-      location.left = elem.x;
-    } else if (elem.offsetParent) {
-      do {
-        location.top += elem.offsetTop;
-        location.left += elem.offsetLeft;
-        elem = elem.offsetParent;
-      } while (elem);
+    if (elem.getBoundingClientRect) {
```

Comment:
@tabanliviu MAINTAINABILITY - No need for the guard here - plus potential for consolidating `getImagesInfoParent` and `getElementTopLeft`.